staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -467,8 +467,24 @@ const (
 	// if a particular node is alive and hence should be available for new
 	// virtual machine instance scheduling. Used on Node.
 	VirtHandlerHeartbeat string = "kubevirt.io/heartbeat"
+	// Namespace recommended by Kubernetes for commonly recognized labels
+	AppLabelPrefix = "app.kubernetes.io"
+	// This label is commonly used by 3rd party management tools to identify
+	// an application's name.
+	AppNameLabel = AppLabelPrefix + "/name"
+	// This label is commonly used by 3rd party management tools to identify
+	// an application's version.
+	AppVersionLabel = AppLabelPrefix + "/version"
+	// This label is commonly used by 3rd party management tools to identify
+	// a higher level application.
+	AppPartOfLabel = AppLabelPrefix + "/part-of"
```

Comment:
should this be `AppProductLabel = AppLabelPrefix + "/product"` I find it strange the terminology is different for this label than for rest of the code.

Reply:
The commonly used label is "app.kubernetes.io/part-of"

I guess it's all a matter of point of view. In this case our components are all from the kubevirt project, but pretend we shipped a postfix database for some reason. The question a user or admin would be asking when looking at a pod named "postfix-XXXXX" is "what is this? where did it come from?" -- so from that point of view the "part of" label makes sense.

As far as the decision to use the name "productName" on the kubevirt resource is the person making the decision at that level is thinking in terms of the bundle of applications they're trying to package together. "this is the ____ product offering".

So it just seemed to make sense. naming it the "AppPartOfLabel" didn't seem to convey as much intuitive meaning. names are hard. What do you think?

Reply:
I didn't realize that the "part-of" label was a standard/community thing. In that case this makes sense and the naming does as well